paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/clone/CloneFileInfo.java

```diff
@@ -53,10 +56,18 @@ public String getTargetIdentifier() {
         return targetIdentifier;
     }
 
+    public FileType getFileType() {
```

Comment:
Can you do this in a separate PR?

Reply:
OK, In the later PR, I will fix the clone action.